pkgs/development/libraries/libgudev/default.nix

```diff
@@ -6,8 +6,10 @@
 , udev
 , glib
 , gobject-introspection
+, withIntrospection ? stdenv.buildPlatform == stdenv.hostPlatform
```

Comment:
The rest of the changes could be removed now.

Reply:
I would prefer that in other libraries like atk and gdk-pixbuf too.

Reply:
Done for libgudev. Will also change this for other libraries.